src/MCPClient/lib/clientScripts/verifyMD5.sh

```diff
@@ -23,6 +23,7 @@
 # @version svn: $Id$
 
 checkMD5NoGui="`dirname $0`/archivematicaCheckMD5NoGUI.sh"
+date_format="%a %b %e %_H.%M.%S %Z %Y"
```

Comment:
Should we use instead something like the following?

```bash
date="$(date -u '+%Y%m%dT%H%M%SZ')" # 20170605T194004Z
```

It is compliant with ISO8601 and measured in UTC.